### DISCUSSION TITLE
Add filter option on join query

### DISCUSSION THREAD

**Discussion by shanmuthu83** [toluaina/pgsync#358] [General]
Hi @toluaina

How to add the filter option in join query in schema.json file

for ex.. under children attribute
  ```
"relationship": {
            "variant": "object",
            "type": "one_to_many",
            "foreign_key": {
              "child": ["id"],
              "parent": ["id" ]
            }
          }
```

```
SELECT 
  * 
FROM 
  table_1 t1 
  LEFT JOIN table_2 t2 ON t1.id = t2.id 
  **AND t2.type = 'customer'**
```

**Comment by toluaina** [comment]
This is probably too difficult and likely to lead to complications.
I'd suggest using the plugin approach described [here](https://github.com/toluaina/pgsync/issues/285)
